src/phantom_token.c

```diff
@@ -389,24 +464,17 @@ static ngx_int_t handler(ngx_http_request_t *request)
     introspection_body->data = introspect_body_data;
     introspection_body->len = ngx_strlen(introspection_body->data);
 
-    introspection_request->request_body = ngx_pcalloc(request->pool, sizeof(ngx_http_request_body_t));
```

Comment:
This was a redundant extra allocation, since introspection_request->request_body is just assigned to introspection_request_body later on.